OpenRA.Mods.Common/Traits/CashTrickler.cs

```diff
@@ -87,7 +90,15 @@ void AddCashTick(Actor self, int amount)
 
 		void ModifyCash(Actor self, Player newOwner, int amount)
 		{
-			amount = resources.ChangeCash(amount);
+			// Should the cash trickler use resource storage
+			if (info.UseStorage)
+			{
+				int initialAmount = resources.Resources;
+				resources.GiveResources(amount);
```

Comment:
I wonder if it would be worth to let `GiveResources` return the amount of cash given similarly to what `ChangeCash` does.

Reply:
Is this a showstopper?

Reply:
I don't think so.

Reply:
It should be good to go.